app/views/movies/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1><%= @movie.title %></h1>
+<h4>Creation Year: <%= @movie.creation_year %></h4>
+<h4>Genre: <%= @movie.genre %></h4>
+<div id="movie-<%= @movie.id %>-actors">
+  <% @actors.order_by_age.each do |actor| %>
+    <h5><%= actor.name %></h5>
+    <h6>Age: <%= actor.age %></h6>
+  <% end %>
+  <h5>Average Age of All Actors: <%= @actors.average_age %></h5>
+</div>
+
+<%= form_with url: "/movie_actors", method: :post do |form| %>
+  <%= form.number_field :actor_id %>
+  <%= form.hidden_field :movie_id, value: @movie.id %>
+  <%= form.submit "Submit" %>
```

Comment:
Great work with this form! The route is restful and the form is implemented correctly. 👍